### PR TITLE
Feature/examples upload

### PR DESCRIPTION
**Pull request by lievenvg**
## PR Checklist

This PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Contributing guidelines](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] A [changelog entry](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/guidelines/CHANGELOG.md) was added

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
...
+- `forms/datepicker` Added documentation.
+- `forms/mask` Added documentation.
+- `forms/range-slider` Added documentation.
+- `forms/timepicker` Added documentation.
+- `forms/upload` Added documentation.
+- `forms/wysiwyg` Added documentation.
+- `layout/cookie-consent` Added documentation.
+- `layout/hero` Added documentation.
+- `layout/pane` Added documentation.
+- `layout/pane` Added documentation.
```
Comment (TomVanDenBroeck) [inline CHANGELOG.md]: double text layout/pane

File: packages/forms/examples/forms/pages/upload/upload.page.ts [hunk 2]
```diff
@@ -1,7 +1,259 @@
...
+export class AppModule {};`;
+	public uploadExampleJS1 = `public dropzone1: UploadOptions = {
+	allowedMimeTypes: ['image/jpeg'],
+	maxFileSize: 10000000,
+	queueLimit: 2,
+	type: 'drop',
+	url: 'api/upload',
+};`;
+	public uploadExampleHTML1 = `<aui-upload [options]="dropzone1" (selectUploadedFiles)="onUpload($event)">
+  <div class="aui-upload-message">
```
Comment (TomVanDenBroeck) [inline packages/forms/examples/forms/pages/upload/upload.page.ts]: indenting correct?

File: packages/forms/examples/forms/pages/upload/upload.page.ts [hunk 3]
```diff
@@ -1,7 +1,259 @@
...
+		autoUpload: true,
+		maxFileSize: 2000000,
+		url: 'api/upload',
+};`;
+	public uploadExampleHTML2 = `<aui-upload [options]="dropzone2" (selectUploadedFiles)="onUpload($event)">
+  <div class="aui-upload-button">
+    Upload button
+  </div>
+</aui-upload>`;
+public uploadExampleJS3 = `public files = [];
```
Comment (TomVanDenBroeck) [inline packages/forms/examples/forms/pages/upload/upload.page.ts]: indenting correct?

File: packages/forms/examples/forms/pages/upload/upload.page.ts [hunk 4]
```diff
@@ -1,7 +1,259 @@
...
+// Pass created options into new instance of Uploader
+public uploader = new Uploader({
+    allowedFileTypes: ['jpg', 'jpeg', 'png'],
+    type: 'drop',
+    url: 'api/upload',
+});
+public showError = false;
+public fileName = '';
+});`;
+public uploadExampleJS4 = `public onQueuedFiles(files: File[]) {
```
Comment (TomVanDenBroeck) [inline packages/forms/examples/forms/pages/upload/upload.page.ts]: indenting correct?

File: packages/forms/examples/forms/pages/upload/upload.page.ts [hunk 5]
```diff
@@ -1,7 +1,259 @@
...
+			if (!files.length) {
+					return;
+			}
+
+			this.queuedFiles = this.queuedFiles.concat(files);
+	}
+
+	public onUploadedFiles(files) {
+		this.uploadedFiles = this.uploadedFiles.concat(files);
+}
```
Comment (TomVanDenBroeck) [inline packages/forms/examples/forms/pages/upload/upload.page.ts]: indenting correct?

File: packages/forms/src/lib/upload/README.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+    type: 'button',
+    allowedFileTypes: ['.jpg', 'jpeg', 'png'],
+    autoUpload: true,
+    maxFileSize: 2000000,
+    url: 'api/upload',
+};
+```
+
+```html
+<div class="u-margin-bottom">
```
Comment (TriangleJuice) [inline packages/forms/src/lib/upload/README.md]: I guess there's something wrong with your code editor. It doesn't seem to complain about the tabs (that should be spaces).

File: packages/forms/src/lib/upload/README.md [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+	this.queuedFiles = this.queuedFiles.concat(files);
+}
+
+public onUploadedFiles(files) {
+this.uploadedFiles = this.uploadedFiles.concat(files);
+}
+
+public onInvalidFiles(errorFiles: InvalidFile[]) {
+	this.invalidFiles = errorFiles;
+	if (errorFiles.length > 0) {
```
Comment (TriangleJuice) [inline packages/forms/src/lib/upload/README.md]: Please fix indenting from here.

File: packages/forms/src/lib/upload/README.md [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+    (err) => {
+        console.log('HTTP Error', err);
+    });
+}
+
+/**
+ * Using a custom upload service
+ */
+
+// public uploadFiles(): void {
```
Comment (TriangleJuice) [inline packages/forms/src/lib/upload/README.md]: Same comment remark as above.

File: packages/forms/src/lib/upload/README.md [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
+// 	}
+// 	this.customService.postFile(this.queuedFiles).subscribe(res => {
+// 		this.result = res;
+// 	}, (error) => {
+// 		console.log(error);
+// 	});
+// }
+```
+
+```scss
```
Comment (TriangleJuice) [inline packages/forms/src/lib/upload/README.md]: Please remove this completely (as it doesn't work in all browsers).

**Comment by lievenvg** [comment]
@TriangleJuice and @TomVanDenBroeck
Added bugfix for compatibility with Core Branding, so please rereview.

**Pull request merged**